src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -172,7 +172,11 @@ object DockerPlugin extends AutoPlugin {
     */
   private final def makeLabel(label: (String, String)): CmdLike = {
     val (variable, value) = label
-    Cmd("LABEL", variable + "=\"" + value.toString + "\"")
+    val valueString = Option(value) match {
+      case Some(v) => v.toString
+      case None => ""
```

Comment:
An empty label seems not to be a good default. IMHO we should throw a
helpful error message here, because something else is configured incorrectly.